packages/flutter/lib/src/widgets/icon.dart

```diff
@@ -122,6 +132,19 @@ class Icon extends StatelessWidget {
     );
   }
 
+  /// Wraps the widget with a Semantics widget if [label] is set.
+  Widget _wrapWithSemantics(Widget widget) {
+    if (label == null) {
+      return widget;
+    }
+    return new Semantics(
+      child: new ExcludeSemantics(
```

Comment:
I think this ExcludeSemantics can now be removed?

Reply:
:flushed:
Thought I did that